src/pages/People.tsx

```diff
@@ -0,0 +1,132 @@
+import { useEffect, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Loader } from '../components/Loader';
+import { getPeople } from '../api';
+import { peopleStore } from '../Store';
+
+export const People: React.FC = () => {
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(false);
+  const { human } = useParams();
+  const { people, setPeople } = peopleStore();
+
+  useEffect(() => {
+    setLoading(true);
+    getPeople()
+      .then(setPeople)
+      .catch(err => {
+        setError(true);
+        Error(err.message);
```

Comment:
```suggestion
```
it does nothing, so you can delete it
you already handle an error (you have `error` state), so no need to `throw new Error` or do something like this

Reply:
It's just a show for the 'do something' error.